joinmarket/maker.py

```diff
@@ -213,7 +213,8 @@ def __init__(self, msgchan, wallet):
         self.active_orders = {}
         self.wallet = wallet
         self.nextoid = -1
-        self.orderlist = self.create_my_orders()
+        if not self.msgchan.newnyms:
+            self.orderlist = self.create_my_orders()
```

Comment:
if create_my_orders() is now called in on_welcome() then there's no need to have it here too